### ISSUE TITLE
Update JOSS instructions for editors

### ISSUE THREAD

**Issue by lwasser** [pyOpenSci/peer-review-guide#75]
given this laster review of pyrolite -- https://github.com/pyOpenSci/software-review/issues/20#issuecomment-640197262 it was mentioned by @arfon that it would be best for us to direct maintainers to JOSS to submit. in their submission we can tell them to mention a review already happened here and was accepted by POS. add documentation in the editor templates about this so each editor can provide instructions to the maintainer at the end of a review.

**Comment by lwasser** [comment]
@arfon I am working on the instructions from this aspect of the review now. I'm so sorry - i've been pinging you directly in issues and had forgot about this from last year. I am working on an editor guide - would language like this work for you:

Question: do you want our reviewers to look at the paper.md file and cross check JOSS scope as well?

```markdown
### Instructions for Submitting to JOSS:
If the package fits within the JOSS Scope, once the package has been approved by pyOpenSci, you can direct the package author to [follow the instructions to submit the package to JOSS](https://joss.readthedocs.io/en/latest/submitting.html). 

* tag the issue `7/under-joss-review`

These instructions loosely include:

1. Login to the JOSS website and fill out the JOSS submission form. When you fill out the form, be sure to mention and link to the approved pyOpenSci review.
2. Wait for a JOSS editor to approve the presubmission

JOSS will accept the pyOpenSci review and direct the author through the processing of checking their paper.md file. Once the package is accepted by JOSS, the author will be instructed to add the JOSS doi badge to their package README file.
Once this has been added and the DOI resolves properly, do the following: 

* tag the issue with `8/joss-review-complete`
* close the issue within the pyOpenSci software-review repo.  
```

**Comment by arfon** [comment]
Thanks @lwasser - these guidelines look good to me.

> Question: do you want our reviewers to look at the paper.md file and cross check JOSS scope as well?

No, I think this is our job to do, although you might want to link to the JOSS docs on [what a paper should contain](https://joss.readthedocs.io/en/latest/submitting.html#what-should-my-paper-contain). Also, one minor change to the second bullet (I've made it bold for clarity):

> 1. Login to the JOSS website and fill out the JOSS submission form. When you fill out the form, be sure to mention and link to the approved pyOpenSci review.
> 2. Wait for a JOSS editor to approve the presubmission **(which includes a scope check)**

What do you think?

**Comment by lwasser** [comment]
@arfon this all looks good. i'll make these change now. 
I have a few more questions

We currently have this language  in our guide: 
```
@ maintainer ... It looks like you would like to submit this package to JOSS. Here are the next steps:

1. Login to the JOSS website and fill out the JOSS submission form. When you fill out the form, be sure to mention and link to the approved pyOpenSci review.
2. Wait for a JOSS editor to approve the presubmission (which includes a scope check)
3. Once the package is approved by JOSS, you will be given instructions by JOSS about updating the citation information in your README file.

<< THIS IS THE LANGUAGE THAT I AM UNSURE OF >>
- [ ] Activate Zenodo watching the repo
- [ ] Tag and create a release so as to create a Zenodo version and DOI
- [ ] Submit to JOSS using the Zenodo DOI. We will tag it for expedited review.
```
Can you please clarify how you use / if you need the zenodo doi? Don't you provide a new doi when the package is accepted that is cross-ref compatible?

I just want to ensure our language is clear. (and i am clear as well on the full process!)

i've also added a little breakout to  make sure we are clear on scope! i don't want people to think that they will automatically be accepted in the case that it is not in scope! I also wonder if we want a line about how packages that have been previously published won't get published either. if you recall we have one package that we had to debate about  that in the end did NOT go to JOSS given a previous citation in another journal.

<img width="823" alt="Screen Shot 2021-05-26 at 6 42 32 PM" src="https://user-images.githubusercontent.com/7649194/119748459-30c8e500-be52-11eb-9484-b5eec56283ee.png">

**Comment by arfon** [comment]
Do you need your authors to create a Zenodo archive for pyOpenSci? If not, feel free to leave off this language about Zenodo as we can handle this with the pyOpenSci package owners.

> i've also added a little breakout to make sure we are clear on scope! i don't want people to think that they will automatically be accepted in the case that it is not in scope! I also wonder if we want a line about how packages that have been previously published won't get published either. if you recall we have one package that we had to debate about that in the end did NOT go to JOSS given a previous citation in another journal.

Right. We could say something like 'Packages that have been previously published elsewhere may not be eligible to published with JOSS'. Or something like that?

**Comment by lwasser** [comment]
@arfon perfect. We are strongly encouraging zenodo. Can you clarify how you handle the zenodo DOI? do you redirect it to a new cross-ref recognized DOI?

We know that some of our maintainers won't be able to use zenodo for various reasons but will encourage everyone who can to use zenodo. If they are moving on to JOSS however and don't have zenodo already setup does it make sense for them to hold off and just use the JOSS awarded DOI assuming they are approved in your process?

Please let me know what makes the most sense here. i just want our documentation to be clear!

**Comment by arfon** [comment]
> We are strongly encouraging zenodo. Can you clarify how you handle the zenodo DOI? do you redirect it to a new cross-ref recognized DOI?

The Zenodo DOI for JOSS is so that there is an (off-GItHub) archive of the software. JOSS links to this in the paper, but the paper itself gets its own DOI (from Crossref).

<img width="1483" alt="Screen_Shot_2021-06-03_at_17_48_49" src="https://user-images.githubusercontent.com/4483/120682146-06031000-c494-11eb-9514-cb9edede161d.png">

> If they are moving on to JOSS however and don't have zenodo already setup does it make sense for them to hold off and just use the JOSS awarded DOI assuming they are approved in your process?

No, because they'll still need to archive the software separately themselves regardless.

**Comment by lwasser** [comment]
got it. Thank you! I understand now @arfon thank you for clarifying this for me. 

ZENODO is used to create an archive which is stored outside of github on zenodo's site. But in  the case of a JOSS review, we don't use the DOI because JOSS will generate a cross ref doi so the authors will be asked to remove the zenodo doi badge and use cross ref doi for citation purposes.  But the archive is still listed via zenodo.

I will make a few more edits to our guide but i think i have all that i need to clarify in the docs!

**Comment by arfon** [comment]
> authors will be asked to remove the zenodo doi badge and use cross ref doi for citation purposes. But the archive is still listed via zenodo.

Authors are welcome to keep this in their READMEs etc. but we'd obviously prefer that they cited their shiny new JOSS paper rather than a Zenodo DOI :-)
